index.html

```diff
@@ -0,0 +1,206 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Alina Sredna Week 1 Project</title>
+    <link href='https://fonts.googleapis.com/css?family=Oswald:400,300|Lato:400,100|Merriweather|Lora:400,700italic,400italic' rel='stylesheet' type='text/css'>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <!-- top of page -->
+     <section class="purple-rectangle"></section>
+     <header class="volume">VOLUME 3-SEPTEMBER 2013</header>
+
+    <!-- CSS OFF -->
+    <header class="cssoff">CSS<br>OFF</header>
+
+    <!-- barbie -->
+    <section class="barbie">
+      <figure>
+        <img class="barbie-pic" src="img/barbie.png" alt="Barbie" title="Barbie">
+      </figure>
+      <article class="barbietext">
+        <header class="box-model-header">
+          <h3><strong><em>Box Model</em></strong></h3>
+        </header>
+        <p>The Barbie doll was invented in 1959 by Ruth Handler (co-founder of Mattel), whose own daughter was called Barbara. Barbie was introduced to the world at the American Toy Fair in New York City. Barbie's job was teenage fashion doll. The full name of the first doll was Barbie Millicent Roberts, from Willows, Wisconsin.</p>
+      </article>
+    </section>
+
+    <!-- positioning relatives  -->
+    <section>
+      <figure class="oldpic">
+        <img src="img/oldpic.png" alt="Old Timey Picture" title="Old Timey Picture">
+      </figure>
+      <article class="relatives">
+        <header>
+          <h6>Feature</h6>
+          <h1>Positioning<br>Relatives</h1>
+        </header>
+        <p class="relatives-p">In 1958, English pictorial photographer Henry Peach Robinson created the world's first photomontage by combining five different negatives to make one complete print of a young girl on her deathbed. "Fading Away" - Robinson's first and most famous composite photo - depicts a young girl dying of consumption and was controversial when it was exhibited, many believing it was not a suitable subject for photography.</p>
+      </article>
+    </section>
+
+    <!-- staying afloat -->
+    <section class="afloat">
+      <article>
+        <header>
+          <h1 class="afloat-h1">Staying afloat</h1>
+        </header>
+        <p class="afloat-p">Lobsterman John Aldridge fell into the shark-infested water of Long Island while his crewmates were asleep. He was kept afloat for more than 12 hours by his rubber boots, which he used as flotation devices. The coast guard rescued him 43 miles south of Montauk where he fell off his vessel.</p>
+      </article>
+      <figure class="bootpic">
```

Comment:
While this is a CSS class for the `bootpic`, it could be argued that this is a general UI pattern that will not always be specific to this specific picture. Using a naming convention that reflects the image pattern is more flexible.